exopy/tasks/configs/base_config_views.enaml

```diff
@@ -34,6 +35,7 @@ enamldef BaseConfigView(Container):
     Field: t_f:
         text := config.task_name
         submit_triggers = ['lost_focus', 'return_pressed', 'auto_sync']
+        background << parse_color("#FFFFFF") if config.name_valid else parse_color("#FF000030")
```

Comment:
I believe that `'white'` and `'red'` would have been fine too (and easier to read).

Reply:
I used some transparency to make the red softer and I used the HTML notation for the white for constency.

Reply:
Ok